### DISCUSSION TITLE
Pull Request Not Showing

### DISCUSSION THREAD

**Discussion by Meteor24** [microsoft/winget-pkgs#15317] [Q&A] [answered]
I had submitted a pull request for a program that I wanted to add to the repo and it said that the checks had been completed and that the request has been merged. Does this mean that the manifest in the pull request will work when i type `winget install AnyBurn` or `winget install AnyBurnSetup` into the command prompt and that it is available to everyone with the winget package manager who wants to install AnyBurn? Heres the link to the pull request in case it helps: https://github.com/microsoft/winget-pkgs/pull/15299

**Comment by iDolmatov** [comment]
Worked.
![image](https://user-images.githubusercontent.com/19782385/120249295-d685b500-c293-11eb-821e-7161264343ff.png)

> Pull Request Not Showing

Need remove `is:pr is:open ` or click [14,659 Closed](https://github.com/microsoft/winget-pkgs/issues?q=is%3Apr+is%3Aclosed).

**Comment by denelon** [comment, marked as answer]
@Meteor24 more work has to happen after a PR is merged. We still need to publish the PreIndexed package, and users need to have their local copy updated. This process from PR submitted to package available to users can take anywhere between a few minutes and a few days to complete.
